proteinworkshop/datasets/pdb_dataset.py

```diff
@@ -29,7 +29,9 @@ def __init__(
         remove_ligands: List[str],
         remove_non_standard_residues: bool,
         remove_pdb_unavailable: bool,
-        split_sizes: List[float],
+        split_ratios: List[float],
```

Comment:
Can we follow the same convention (`train_val_test`) https://github.com/a-r-j/ProteinWorkshop/blob/716a1b2e9f5115950946f8d980fc4800b4bf167c/proteinworkshop/datasets/igfold.py#L53

Reply:
done